### PR TITLE
Make username validation cleaner; add highlighting for newly added users

### PR DESCRIPTION
**Pull request by tliu22**
-When hovering over usernames in autocomplete dropdown list, they now auto-validate
-Clear username field after entering
-Add highlighting for newly added users
-Move styling to static/style.css

### CODE REVIEW

**Comment by tliu22** [comment]
We might want to start a .css file?

**Comment by seanlobo** [comment]
Probably better to split it up into a .css file now. Could we do that in this pr?

**Comment by seanlobo** [comment]
Probably better to make a separate .css file now, can you add it in?

File: static/timeseries.js [hunk 1]
```diff
@@ -128,17 +127,18 @@ function sortCharts() {
 
 function addUserEntry(user) {
   if (isStaff(user) && !chartContains(user)) {
+    $userEntry = $("<li>", {
```
Comment (seanlobo) [inline static/timeseries.js]: missing a `var`

File: static/timeseries.js [hunk 2]
```diff
@@ -149,6 +149,11 @@ function addUserEntry(user) {
           })
         )
     );
+    $('#addStaff').val('');
+    sortUserEntries();
+    $userEntry.effect("highlight", {
+      color: "#5cb85c"
+    }, 1000)
```
Comment (seanlobo) [inline static/timeseries.js]: Can we increase this to `2000`? It seems pretty short right now.

**Pull request merged**